dask/array/tests/test_array_core.py

```diff
@@ -130,6 +130,57 @@ def test_blockwise_literals():
     assert_eq(z, x)
 
 
+def test_blockwise_1_in_shape_I():
```

Comment:
@mrocklin can you confirm that these tests look correct to you? I still struggle with the blockwise syntax.

Reply:
I can also try to be clearer and in `test_f` make sure that the shape of `a` and `b` are compatible and can be broadcasted together.

Reply:
Sorry for the delay here.  I probably won't get to this until this weekend.